File: backends/php-laravel/app/Infrastructure/Persistence/Eloquent/Models/Staff.php
<?php
/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */
namespace App\Infrastructure\Persistence\Eloquent\Models;

use App\Support\Models\AppMasterModel;
use Database\Factories\StaffFactory;

/**
 * スタッフModelクラスです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 * @package App\Infrastructure\Persistence\Eloquent\Models
 */
class Staff extends AppMasterModel
{
    protected $table = 'staffs';

    protected $guarded = [];

    /**
     * {@inheritdoc}
     */
    protected static function newFactory()
    {
        return StaffFactory::new();
    }
}

File: backends/ruby-rails/app/infrastructure/model/staff.rb
# frozen_string_literal: true
#
# スタッフモデルを定義するインフラストラクチャモジュール。
#
# @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>

module Infrastructure
  module Model
    # staffs テーブルに対応する ActiveRecord モデルです。
    class Staff < ActiveRecord::Base
      self.table_name = "staffs"
    end
  end
end
